core/src/test/java/com/graphhopper/routing/DirectedBidirectionalDijkstraTest.java

```diff
@@ -250,11 +250,11 @@ public void directedRouting() {
         int leftNorth = graph.edge(9, 0, 1, true).getEdge();
 
         // make paths fully deterministic by applying some turn costs at junction node 2
-        addTurnCost(7, 2, 3, 1);
-        addTurnCost(7, 2, 6, 3);
-        addTurnCost(1, 2, 3, 5);
-        addTurnCost(1, 2, 6, 7);
-        addTurnCost(1, 2, 7, 9);
+        setTurnCost(7, 2, 3, 1);
```

Comment:
I think its better to do such renamings in a separate commit in master (same with turn cost extension/storage). I changed this in master and merged here again

Reply:
Yes, you are right. Thanks!